documentation/modules/auxiliary/scanner/afp/afp_login.md

```diff
@@ -0,0 +1,45 @@
+## Vulnerable Application
+
+Apple Filing Protocol (AFP) is Apple's file sharing protocol similar to SMB, and NFS. This module attempts to bruteforce authentication credentials for AFP.
+
+References:
+https://developer.apple.com/library/mac/documentation/Networking/Reference/AFP_Reference/Reference/reference.html
+
+https://developer.apple.com/library/mac/documentation/networking/conceptual/afp/AFPSecurity/AFPSecurity.html
+
+1. `sudo apt-get install netatalk`
```

Comment:
I would add a header, this is debian/ubuntu based, but there may be a specific difference.

Add something like: ```### Debian 19``` or ```### Kali 2019.3```